backend/dropbox/dropbox.go

```diff
@@ -235,6 +243,31 @@ func NewFs(name, root string, m configmap.Mapper) (fs.Fs, error) {
 		Client:          oAuthClient,    // maybe???
 		HeaderGenerator: f.headerGenerator,
 	}
+
+  // NOTE: needs to be created pre-impersonation so we can look up the impersonated user
+	f.team = team.New(config)
+
+	if opt.Impersonate != "" {
+
+		user := team.UserSelectorArg{
+			Email: opt.Impersonate,
+		}
+        user.Tag = "email"
+
+		members := []*team.UserSelectorArg{&user}
+		args := team.NewMembersGetInfoArgs(members)
+
+		member_ids, err := f.team.MembersGetInfo(args)
+		
+		if err != nil {
+			return nil, errors.Errorf("Invalid Dropbox Team Member: %s", opt.Impersonate)
```

Comment:
This should be

```go
return nil, errors.Wrapf(err, "invalid dropbox team member: %q", opt.Impersonate)
```

Then it will return the original error to the user with your annotation.   According to the go style guide these should start with small letters too!  I'd use %q to put the string in "quotes" also.

Reply:
adjusted this, as well as changed `member_ids` to `memberIds`  per styleguide